src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1637,6 +1647,7 @@ void goto_instrument_parse_optionst::help()
     HELP_REMOVE_CONST_FUNCTION_POINTERS
     " --add-library                add models of C library functions\n"
     " --model-argc-argv <n>        model up to <n> command line arguments\n"
+    " --remove-function-body <f>   remove the implementation of function <f>\n"
```

Comment:
Per [the coding standard](https://github.com/diffblue/cbmc/blob/master/CODING_STANDARD#L112), it would be good if this flag could be moved to the code that implements it, unless this really can only be used in `goto-instrument`?